data.json

```diff
@@ -1654,5 +1654,11 @@
     "githubUsername": "AnishDe12020",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png",
     "issueNumber": 870
-  }
+  },
+  {
+    "name": "Raghav Maskara",
+    "githubUsername": "RaghavMaskara21",
+    "imageUrl": "https://user-images.githubusercontent.com/79410102/143765055-791a42ab-2cc6-4c9d-9e45-6529eb147aa4.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/143782190-a08fdf59-6943-4538-874b-58276d7f7f3f.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/143782190-a08fdf59-6943-4538-874b-58276d7f7f3f.png",
```